combustion-android-ble/src/main/java/inc/combustion/framework/ble/ProbeManager.kt

```diff
@@ -622,6 +621,7 @@ internal class ProbeManager(
         _probe.value = _probe.value.copy(
             baseDevice = _probe.value.baseDevice.copy(
                 connectionState = connectionState,
+                fwVersion = fwVersion,
```

Comment:
It seems like this is the right approach, though if it was omitted in the past, it's not clear why, and the comment reflects the new implementation.